index.html

```diff
@@ -38,6 +38,8 @@ <h1>Getting Started with JavaFX 11</h1>
                         <a class="list-group-item" href="#install-javafx">Run HelloWorld using JavaFX 11</a>
                         <a class="list-group-item" href="#maven">Run HelloWorld via Maven</a>
                         <a class="list-group-item" href="#gradle">Run HelloWorld via Gradle</a>
+                        <a class="list-group-item" href="#fat-jar">Creating a fat Jar</a>
```

Comment:
Probably better to switch those? (First do it the right way (modular app), then the old way (fat jar))